proposals/table-component.md

```diff
@@ -0,0 +1,221 @@
+# Table Component
+
+Component to render tabular data.
+
+## Features
+
+* Sorting
+
+## Currently Unsupported
+
+* Pagination
```

Comment:
pagination is possible with the current api, see the second 'advanced' example. it would not be a tight ui integration at the moment. 

unifying the pagination story should only include new properties

Reply:
cool, thanks!

I meant the whole `Currently unsupported` section - does it apply to them too?

Reply:
not that i can think of